src/foam/u2/view/RadioView.js

```diff
@@ -20,56 +20,23 @@ foam.CLASS({
   name: 'RadioView',
   extends: 'foam.u2.view.ChoiceView',
 
-  requires: [ 'foam.u2.DisplayMode' ],
+  requires: [ 'foam.u2.DisplayMode', 'foam.u2.view.RadioButton' ],
 
+  imports: ['theme'],
```

Comment:
Should be optional: 'theme?'

Reply:
We can remove this import as we only use theme in the button